helm-charts/0.24.5/charts/spire/templates/spire-controller-manager-config.yaml

```diff
@@ -19,8 +19,8 @@ data:
     kind: ControllerManagerConfig
     metrics:
       bindAddress: 127.0.0.1:8082
-    healthProbe:
-      bindAddress: 127.0.0.1:8083
+    health:
+      healthProbeBindAddress: 0.0.0.0:8083
```

Comment:
Not sure about the 0.0.0.0 binding; I’l take a note to test it with 127.0.0.1 too